src/ScheduleFunctions.cpp

```diff
@@ -1098,6 +1098,7 @@ void validate_schedule(Function f, Stmt s, const Target &target, bool is_output,
         }
     }
 
+#if 0
```

Comment:
Fixed - this was actually a workaround for an unrelated, now-fixed problem...thanks for catching this one!